types/pump/pump-tests.ts

```diff
@@ -52,3 +53,16 @@ pump(createReadStream('/dev/random'), toHex(), createWriteStream('/dev/null'));
 
 // $ExpectType Stream
 pump([createReadStream('/dev/random'), toHex(), createWriteStream('/dev/null')]);
+
```

Comment:
There is no reason to have these tests depend on Gulp. Please use use `fs` streams to make sure a callback with an optional error is type correct given your type addition.

Reply:
> There is no reason to have these tests depend on Gulp.

But that's how I use `pump`. And https://github.com/DefinitelyTyped/DefinitelyTyped#my-package-teststs says that the tests should show how the API is being used. And @typescript-bot says there's no problem with test performance.

But in the interests of getting this PR merged, and since I'm not expecting code breakage anytime soon, I've removed the dependency on Gulp as requested 8f1cf7a.

Reply:
I get that you use pump with Gulp, but the thing you want to illustrate is that the library works with streams. It just so happens Gulp is stream based, but we don't wanna make these tests depend on Gulp's interface. Thank you for making this changes.